warehouse/models/mart/gtfs/dim_stop_times.sql

```diff
@@ -31,9 +43,13 @@ dim_stop_times AS (
         continuous_drop_off,
         shape_dist_traveled,
         timepoint,
+        COALESCE(warning_duplicate_primary_key, FALSE) AS warning_duplicate_primary_key,
```

Comment:
nit: I think you can do this with a window function and without the CTE?

using produce from https://cloud.google.com/bigquery/docs/reference/standard-sql/window-function-calls#produce_table

```
WITH Produce AS
         (SELECT 'kale' as item, 23 as purchases, 'vegetable' as category
          UNION ALL
          SELECT 'banana', 2, 'fruit'
          UNION ALL
          SELECT 'cabbage', 9, 'vegetable'
          UNION ALL
          SELECT 'apple', 8, 'fruit'
          UNION ALL
          SELECT 'leek', 2, 'vegetable'
          UNION ALL
          SELECT 'lettuce', 10, 'vegetable')
SELECT *
     , COUNT(*) OVER (PARTITION BY category) > 1 AS warning_duplicate_primary_key
FROM Produce;

```